scripts/integration_test.py

```
@@ -0,0 +1,286 @@
+    print("=" * 60)
+    print("AI Gateway Integration Test")
+    print("=" * 60)
+    print(f"Azure OpenAI endpoint:      {endpoint}")
+    key_fp = hashlib.sha256(api_key.encode()).hexdigest()[:12]
+    print(f"Azure OpenAI API key:       sha256:{key_fp}")
+    if emb_endpoint != endpoint:
+        print(f"Embedding endpoint:         {emb_endpoint}")
+        emb_key_fp = hashlib.sha256(emb_api_key.encode()).hexdigest()[:12]
+        print(f"Embedding API key:          sha256:{emb_key_fp}")
+    else:
+        print(f"Embedding endpoint:         (same as main)")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)